app/Controllers/feedController.php

```diff
@@ -427,7 +427,7 @@ public static function actualizeFeed($feed_id, $feed_url, $force, $simplePiePush
 				$entryDAO->commit();
 			}
 
-			if ($pubSubHubbubEnabled && $feed->hubUrl() && $feed->selfUrl()) {	//selfUrl has priority for WebSub
+			if ($pubsubhubbubEnabledGeneral && $feed->hubUrl() && $feed->selfUrl()) {	//selfUrl has priority for WebSub
```

Comment:
Example: We register to `https://push-pub.appspot.com/feed` which `self` is `http://push-pub.appspot.com/feed`